src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Making your website "cross-origin isolated" using COOP and COEP
+subhead: >
+  Use COOP and COEP to set up a cross-origin isolated environment and enable
+  powerful features like `SharedArrayBuffer`, `performance.measureMemory()` or
+  the JS Self-Profiling API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Use COOP and COEP to set up such an environment and
+  enable powerful features like `SharedArrayBuffer`,
+  `performance.measureMemory()` or the JS Self-Profiling API.
+authors:
+  - agektmr
+hero: hero.png
+date: 2020-04-13
+updated: 2020-04-14
+tags:
+  - post
+  - security
+---
+Some web APIs increase the risk of side-channel attacks like Spectre. To 
+mitigate that risk, browsers offer an opt-in-based isolated environment called 
+cross-origin isolated. With a cross-origin isolated state, the webpage will be 
+able to use privileged features including:
+
+* [`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
```

Comment:
Please check this messaging with @lutzvahl

Reply:
To name SABs is fine for me as we're not talking about Chrome.  But we might remove "SharedArrayBuffer (required for WebAssembly Threads)" from the [Conclusion](https://deploy-preview-2545--web-dev-staging.netlify.com/coop-coep/#conclusion) section as we're talking about upcoming Chrome releases. @natorion WDYT?

Reply:
Removed the mention about SharedArrayBuffer at the conclusion part.